front/src/containers/SearchPage/SearchView2.tsx

```diff
@@ -891,7 +902,8 @@ class SearchView2 extends React.Component<SearchView2Props, SearchView2State> {
             key="default"
             id="dropdown-basic-default"
             style={{
-              margin: '1em 1em 1em 0',
+              margin: '0em 1em 0em 1em',
+              width: "200px",
```

Comment:
Looks like we're using single quotes for styles so would recommend keeping it consistent.